tests/prague/eip7692_eof_v1/tracker.md

```diff
@@ -379,25 +379,25 @@
 - [x] EXTDELEGATECALL from EOF to legacy fails (evmone-tests: state_tests/state_transition/eof_calls/eof1_extdelegatecall_legacy.json)
 - [ ] EXTSTATICCALL forwards static mode (evmone-tests: state_tests/state_transition/eof_calls/extdelegatecall_static.json)
 - [x] EXTCALL with value success (evmone-tests: state_tests/state_transition/eof_calls/extcall_with_value.json)
-- [ ] EXTCALL with value from EXTSTATICCALL (evmone-tests: state_tests/state_transition/eof_calls/extcall_static_with_value.json)
+- [x] EXTCALL with value from EXTSTATICCALL (evmone-tests: state_tests/state_transition/eof_calls/extcall_static_with_value.json)
```

Comment:
I don't like having paths on external references in this file when the test is marked as implemented, because it has happened in the past that this link becomes stale if the other repo is updated.

Also this seems to suggest that the tests marked as [x] will not be implemented in this repo.

Reply:
@marioevz whoopsie, I think I totally missed this comment, sorry.

So I guess in order to avoid stale links, I should remove the evmone-tests links for the items I gave an [x]?

Though I'm not sure I share your concerns, hopefully we're done migrating at one point and we can just drop this file completely (before those links go stale)? I thought the tests with [x] mean just that they're taken care of in EEST.

Reply:
Hey @marioevz I'm not sure how to proceed here. The items marked as [x] in this PR are not the only ones marked [x] and with the link left, so I don't want to make things inconsistent in the file. Maybe we should decide how we want to proceed here and potentially fix in bulk in a separate PR?

Reply:
ah ok, I thouht you've meant remove altogether. OK I'll put the EEST links in then

Reply:
@marioevz See commit `Update tracker.md with implemented EEST tests links`